posthog/hogql/modifiers.py

```diff
@@ -44,6 +45,9 @@ def set_default_modifier_values(modifiers: HogQLQueryModifiers, team: "Team"):
     if modifiers.materializationMode is None or modifiers.materializationMode == MaterializationMode.auto:
         modifiers.materializationMode = MaterializationMode.legacy_null_as_null
 
+    if modifiers.optimizeJoinedFilters is None:
+        modifiers.optimizeJoinedFilters = OptimizeJoinedFilters.false
```

Comment:
I know you're doing this so that you can release and test on prod data - but, is there a scenario in which we'd intentionally set this to `false` once fully released?

Reply:
Yeah, if we discover that something went catastrophically wrong for a team one month from now, we can quickly disable it for them, etc. I imagine in the future we'll have multiple different filters we can join and possibly opt out of. Plus this makes it easy to plug the optimization into a test and check the before/after.